google-cloud-storage/acceptance/storage/bucket_generate_signed_post_policy_v4_test.rb

```diff
@@ -61,6 +62,62 @@
     end
   end
 
+  it "generates a signed post object v4 using signBlob API" do
+    iam_credentials_client = Google::Apis::IamcredentialsV1::IAMCredentialsService.new
+    # Get the environment configured authorization
+    scopes =  ['https://www.googleapis.com/auth/cloud-platform']
+    iam_credentials_client.authorization = Google::Auth.get_application_default(scopes)
```

Comment:
Please drop the parens. Similar for the other new acceptance tests below. Also similar for the inline examples further below.